mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -287,15 +408,26 @@ class DistributedLDAModel private (
     private[clustering] val globalTopicTotals: LDA.TopicCounts,
     val k: Int,
     val vocabSize: Int,
-    private[clustering] val docConcentration: Double,
-    private[clustering] val topicConcentration: Double,
+    protected[clustering] val docConcentration: Vector,
```

Comment:
For now, shall we add a check in the constructor to make sure docConcentration is valid (1 value, or k copies of the same value)?